docs/dev/guides/gvisor.rst

```diff
@@ -0,0 +1,52 @@
+gVisor Installation
+===================
+
+You can mostly get by just following installation instructions in the `gVisor
+Docker Quick Start`_ guide.
+
+There are a few caveats to installation, which likely depend on your local
+environment. For ``systemd`` based OS, you do need to configure the Docker
+daemon to avoid systemd cgroups.
+
+.. tabs::
+
+    .. tab:: Arch
+
+        Follow the installation and quick start directions like normal:
+
+        .. code:: console
+
+            % yay -S gvisor-bin
+            ...
+            % sudo runsc install
+
+        You do need to instruct Docker to avoid systemd cgroups. You will need
+        to make further changes to ``/etc/docker/daemon.json``:
+
+        .. code:: json
+
+            {
+                "runtimes": {
+                    "runsc": {
+                        "path": "/usr/bin/runsc"
+                    }
+                },
+                "exec-opts": ["native.cgroupdriver=cgroupfs"]
+            }
+
+Docker is correctly configured when you can run this command from the quick
+start guide:
```

Comment:
Just note that you have to restart docker at this point. Otherwise, the runtime is unknown: `sudo systemctl restart docker`

Reply:
It's noted in the guide, but feels worth duplicating here